Fix logo color, hero icons sizes and gap

Fatemeh9595 opened the pull request:
-The BlueHouse logo supposed to be blue and 176*64 just like figma .
-The distances of the icons(clock , money , house) from their text and accomodation options get fixed. 
-but there is something about these icons sizes , in the figma the size of them is 34*34, then to not to break the "breakpoints" i did a simple calculation that these icons are 72% their container(which is 48*48) , so if the size of container changed in different breakpoints , it can change the icon sizes too .

Fatemeh9595 committed:
BlueHouse Logo changed from black to blue

Fatemeh9595 committed:
fixing the icon's size and gap under carousel

Fatemeh9595 committed:
fixing the distance between Hero and accomodation options

Siciliamia1 commented on src/components/OurServices/OurServices.module.scss:
Check this video that shows what happens to the icons when you go below the desktop breakpoint with the changes you made to the rule. It changes the icons size of the mobile version too


https://github.com/user-attachments/assets/165a6d4d-2b11-4aff-8ced-8aaf83c29722

Siciliamia1 commented on src/components/OurServices/OurServices.jsx:
in line 44 you changed the gap but the result is still a big gap, in line 51-52 try changing the value if it IS desktop, maybe  that can normalize the gap size

Fatemeh9595 committed:
Revert "fixing the icon's size and gap under carousel"

This reverts commit c9e5f2898a114526bf3a63e71edbb8a2055dc73b.

Fatemeh9595 commented:
Dear supervisor, I’ve made the new changes. Please review again when you have time. here i will explain what i did exactly : 
since apparently there were no problems with the commits "Logo" and "the gap between hero and accomodation options" , i just left them like that , but i reverted a comment called "the icon's sizes and gap" , and i did some changes respect what you suggested me : 
1- i removed the previous logic that was given to the "imageContainer's" and added the fixed height and width of 34.3px as it was in Figma
2- i changed the gap of "s.serviceCard" to 14.84px just like it was in Figma

Siciliamia1 reviewed (commented):
Reviewing the icons sizes, im afraid they are still off. The Money icon has missing parts and the houses are smaller compared to the others according to the Figma mockup

Current:
<img width="1276" height="105" alt="image" src="https://github.com/user-attachments/assets/b016e487-3469-46f9-9dc6-722900b98a61" />
Figma: 
<img width="763" height="54" alt="image" src="https://github.com/user-attachments/assets/df519ef2-4268-4a51-b6f2-7989cabae244" />

Fatemeh9595 committed:
Revert "fixing the icon's size and gap under carousel"

This reverts commit c9e5f2898a114526bf3a63e71edbb8a2055dc73b.

Fatemeh9595 commented:
Dear supervisor , i just finished refixing the icons sizes.
